private/buf/bufgen/internal/bufgenv2/file_option.go

```diff
@@ -19,131 +19,203 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+
+	"github.com/bufbuild/buf/private/bufpkg/bufimage/bufimagemodify/bufimagemodifyv2"
+	"google.golang.org/protobuf/types/descriptorpb"
 )
 
 const (
-	// FileOptionJavaPackage is the file option java_package.
-	FileOptionJavaPackage FileOption = iota + 1
-	// FileOptionJavaOuterClassname is the file option java_outer_classname.
-	FileOptionJavaOuterClassname
-	// FileOptionJavaMultipleFiles is the file option java_multiple_files.
-	FileOptionJavaMultipleFiles
-	// FileOptionJavaStringCheckUtf8 is the file option java_string_check_utf8.
-	FileOptionJavaStringCheckUtf8
-	// FileOptionOptimizeFor is the file option optimize_for.
-	FileOptionOptimizeFor
-	// FileOptionGoPackage is the file option go_package.
-	FileOptionGoPackage
-	// FileOptionCcEnableArenas is the file option cc_enable_arenas.
-	FileOptionCcEnableArenas
-	// FileOptionObjcClassPrefix is the file option objc_class_prefix.
-	FileOptionObjcClassPrefix
-	// FileOptionCsharpNamespace is the file option csharp_namespace.
-	FileOptionCsharpNamespace
-	// FileOptionPhpNamespace is the file option php_namespace.
-	FileOptionPhpNamespace
-	// FileOptionPhpMetadataNamespace is the file option php_metadata_namespace.
-	FileOptionPhpMetadataNamespace
-	// FileOptionRubyPackage is the file option ruby_package.
-	FileOptionRubyPackage
+	// fileOptionJavaPackage is the file option java_package.
+	fileOptionJavaPackage fileOption = iota + 1
```

Comment:
why does this still exist?

Reply:
This is to keep allowing per-file overrides from v1

Reply:
Why is v1 affecting v2? I'm not sure I understand

Reply:
This option will be removed, as per comment above